src/main/scala/org/allenai/plugins/DeployPlugin.scala

```diff
@@ -103,6 +103,8 @@ object DeployPlugin extends AutoPlugin {
     // Clients may override to perform a different action in addition to staging (i.e. running
     // database migrations).
     val preDeploy = taskKey[Unit]("Prep the project for a deploy. Defaults to `stageAndCacheKey`.")
+
+    val deployNpmBuild = taskKey[Unit]("Runs the npm build")
```

Comment:
I don't love the naming, but I think it helps clarify that this is in the deploy plugin, so it's probably fine to leave as-is.